courses/free-courses-te.md

```diff
@@ -0,0 +1,206 @@
+# Index
+
+* [Android](#android)
+* [Angular](#angular)
+* [Artificial Intelligence](#artificial-intelligence)
+* [Automata Theory](#automata-theory)
+* [Bash and Shell](#bash-and-shell)
+* [C](#c)
+* [C++](#c++)
+* [Cloud Computing](#cloud-computing)
+* [Compiler Design](#compiler-design)
+* [CSS](#css) 
+* [Database Management Systems](#database-management-systems)
+* [Data Science](#data-science)
+* [Data Structures](#data-structures)
+* [Deep Learning](#deep-learning)
+* [Ethical Hacking](#ethical-hacking)
+* [ExpressJS](#expressjs)
+* [HTML](#html)
+* [Java](#java)
+* [Javascript](#javascript)
+* [Laravel](#laravel)
+* [Machine Learning](#machine-learning)
+* [MongoDB](#mongodb)
+* [NextJS](#nextjs)
+* [NodeJS](#nodejs)
+* [PHP](#php)
+* [Python](#python)
+* [ReactJS](#reactjs)
+* [SQL](#sql)
+* [VueJs](#vuejs)
+
+
+## NPTEL
+
+* [Various Courses in Telugu](https://sites.google.com/nptel.iitm.ac.in/translated-ebook/telugu)
+
+
+### Android
+
+* [Android App Dev](https://www.youtube.com/watch?v=7ZLKpN8vXLo) - Python Life
+* [Android Playlist](https://www.youtube.com/playlist?list=PLeCpoxUq3EhmgHp7KI-Ih3X2PcNYRtC5y) - Sai Gopi
+
+
+### Angular
+
+* [Angular For Beginners](https://www.youtube.com/watch?v=9MxS8oNlnMM) - TeluguTechSteps
```

Comment:
Don't invent titles! Use the title given by the resource. You can edit, but leave at least the first part of the title.

Reply:
Ok got it and on it 💯